nix/sources.json

```diff
@@ -55,16 +55,16 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
```

Comment:
`xrefcheck` is mentioned twice in `pipeline.yaml`, let's have a look:

-----

  - label: crossref-verify
    if: *not_scheduled_autodoc
    commands:
    - nix run -f ci.nix xrefcheck -c
        xrefcheck --mode local-only --config ./.crossref-verifier.yaml
    soft_fail: true # TODO: remove
```

* The file `/.crossref-verifier.yaml` only contains default values, so we can delete it
* `--mode local-only` is not needed either`
* The CI step is named `crossref-verify`. This is the ooooold name of `xrefcheck` :P It has been renamed to "xrefcheck" a long time ago - so let's replace all references of the name "crossref-verify" with "xrefcheck"
* Let's remover `soft_fail: true`. The CI step is [in fact failing](https://buildkite.com/serokell/tezos-btc/builds/1081#01845296-d3bd-4a04-b3e3-5c80b5a0cd09), but we're not being told about it, so it goes unnoticed.

----

```
  - label: crossref-verify generated doc
    key: crossref-verify-generated
    if: *not_scheduled_autodoc
    depends_on:
      - contract-doc
      - contract-doc-master
    commands:
    - mkdir tmp
    - if [ "$BUILDKITE_BRANCH" = "master" ];
      then CONTRACT_DOC_STEP="contract-doc (master)";
      else CONTRACT_DOC_STEP="contract-doc";
      fi
    - buildkite-agent artifact download TZBTC-contract.md tmp/ --step "$$CONTRACT_DOC_STEP"
    - nix run -f ci.nix xrefcheck -c
        xrefcheck --mode local-only --config ./.crossref-verifier.yaml --root tmp
    soft_fail: true # TODO: remove
```

Basically, same items as above.

Reply:
implemented the mentioned fixes and fixed the broken links (in a second commit) as well.